### PR TITLE
Publish descriptions

### LINKED ISSUE #736: Publish descriptions
In review we have started adding descriptions to new features as a matter of course. Let's add the missing ones and then add this to the schema and published data.

### PR DESCRIPTION
**Pull request by captainbrosset**
This PR makes the description field mandatory. Fixes #736.

The PR will fail until the following PRs are merged: #750, #751, #752, #753, #754, #755, #756, #757, #758, #759, #760, #862.

### CODE REVIEW

**Review by ddbeck — APPROVED** [review]

File: schemas/defs.schema.json
Comment (foolip) [inline schemas/defs.schema.json:48]: What should we say about the Markdown syntax inside these strings? Since BCD has HTML in notes, my suggestion is to do the same: https://github.com/web-platform-dx/web-features/pull/882

**Comment by foolip** [comment]
@ddbeck, in the interest of getting this done I've merged some descriptions without waiting for your review. You most likely have some tweaks you might want to make however, so can you review these before we call this issue resolved?

- https://github.com/web-platform-dx/web-features/pull/754
- https://github.com/web-platform-dx/web-features/pull/755
- https://github.com/web-platform-dx/web-features/pull/760

The three remaining PRs are:

- https://github.com/web-platform-dx/web-features/pull/757
- https://github.com/web-platform-dx/web-features/pull/758
- https://github.com/web-platform-dx/web-features/pull/759

**Comment by foolip** [comment]
@captainbrosset all of your PRs are merged now, can you rebase this and see if any new features are missing descriptions?

**Comment by captainbrosset** [comment]
I think we still want https://github.com/web-platform-dx/web-features/pull/862

**Comment by foolip** [comment]
That's now merged too.

**Comment by captainbrosset** [comment]
@ddbeck @foolip all tests now pass! Thanks so much for the reviews on those new descriptions. I'll let you decide whether this can be merged as is, or if it needs more changes.

**Comment by foolip** [comment]
@captainbrosset I've sent https://github.com/web-platform-dx/web-features/pull/898 so we can lock in the wins while https://github.com/web-platform-dx/web-features/pull/882 is in flight. Can you revert that change in this PR after it's merged?

**Comment by captainbrosset** [comment]
> @captainbrosset I've sent #898 so we can lock in the wins while #882 is in flight. Can you revert that change in this PR after it's merged?

I see, so remove the TODO comment and `if` block from 898 after it merges?

**Comment by foolip** [comment]
@captainbrosset yes, exactly.

**Comment by captainbrosset** [comment]
@foolip `if` block removed. Take a final look before we merge?

**Comment by foolip** [comment]
@captainbrosset in order to not change what's in the `description` field soon after we publish it, I think we should land https://github.com/web-platform-dx/web-features/pull/882. Feel free to review that if you have some time for it.

**Comment by foolip** [comment]
@captainbrosset can you update the schema to include `description_html`?

**Pull request merged**
